CMakeLists.txt

```diff
@@ -135,8 +136,8 @@ set(LUAJIT_INCLUDE "${LUAJIT_SRC}")
 set(LUAJIT_LIB "${LUAJIT_SRC}/libluajit.a")
 ExternalProject_Add(
   luajit
-  URL "https://s3.amazonaws.com/download.draios.com/dependencies/LuaJIT-2.0.3.tar.gz"
-  URL_HASH "SHA256=55be6cb2d101ed38acca32c5b1f99ae345904b365b642203194c585d27bebd79"
+  URL "https://github.com/LuaJIT/LuaJIT/archive/v2.0.3.tar.gz"
```

Comment:
Just checked the differences that are making the sha different here, it's just that the version from github contains the gitignore files

luajit1 is the one from s3, luajit2 is the one from github
```
diff -qr luajit1/LuaJIT-2.0.3/ luajit2/LuaJIT-2.0.3
Only in luajit2/LuaJIT-2.0.3: .gitignore
Only in luajit2/LuaJIT-2.0.3/src: .gitignore
Only in luajit2/LuaJIT-2.0.3/src/host: .gitignore
Only in luajit2/LuaJIT-2.0.3/src/jit: .gitignore
```

Reply:
👍